src/javascript/ban.md

```diff
@@ -0,0 +1,43 @@
+# ban
+Bans an user from the current guild.
+
+## Syntax
+```js
+ban(userID)
+```
+
+### Parameters
+- `userID` : The user to ban. Value must be a valid user snowflake ID.
+
+### Permissions
+Required permissions which the bot must have for this function to work properly.
```

Comment:
I'd like to get a suggestion, whether this statement is good or need some change.

Reply:
`readmessages` is not really needed for this specific function